frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -40,12 +40,20 @@ const initialQuery: IPropertyFilter = {
   agencies: [],
 };
 
-const getServerQuery = (state: {
-  pageIndex: number;
-  pageSize: number;
-  filter: IFilterBarState;
-  agencyIds: number[];
-}) => {
+/**
+ * Get the server query
+ * @param ignorePropType - Whether to ignore the PropertyTypes (for doing excel export with all values)
+ * @param state - Table state
+ */
```

Comment:
Did I do this right?

Reply:
You can test by hovering over code that calls `getServerQuery`.  It should show your description.